include/z64item.h

```diff
@@ -2,12 +2,74 @@
 #define Z64ITEM_H
 
 typedef enum {
-    /* 0x00 */ EQUIP_SWORD,
-    /* 0x01 */ EQUIP_SHIELD,
-    /* 0x02 */ EQUIP_TUNIC,
-    /* 0x03 */ EQUIP_BOOTS
+    /* 0 */ EQUIP_TYPE_SWORD,
+    /* 1 */ EQUIP_TYPE_SHIELD,
+    /* 2 */ EQUIP_TYPE_TUNIC,
+    /* 3 */ EQUIP_TYPE_BOOTS,
+    /* 4 */ EQUIP_TYPE_MAX
 } EquipmentType;
 
+// `EquipInv*` enums are for Inventory#equipment (for example `CHECK_OWNED_EQUIP` macro)
```

Comment:
What does this # notation mean? The only place I've seen it used in C is for stringification.

Reply:
In some documentation syntax (like javadoc) it's a way to reference a field method or something inside a class
Here `Inventory#equipment` means the member `equipment` of the struct `Inventory`

But I should get in the habit of using `.` instead, I'll change it

Reply:
Ah, I see, thanks. I *suppose* the C equivalent is `::`. Or at least, that's what VSCode uses, even though it's actually C++ syntax.